lambdas/data-migration2/tests/test-granulesAndFiles.js

```diff
@@ -532,6 +533,104 @@ test.serial('migrateGranuleRecord updates an already migrated record if the upda
   t.deepEqual(record.updated_at, new Date(newerGranule.updatedAt));
 });
 
+test.serial('migrateGranuleRecord updates an already migrated record if migrateAndOverwrite is set to "true"', async (t) => {
+  const {
+    knex,
+    granulePgModel,
+    testCollection,
+    testExecution,
+  } = t.context;
+
+  const testGranule = generateTestGranule({
+    collectionId: constructCollectionId(testCollection.name, testCollection.version),
+    execution: testExecution.url,
+    status: 'completed',
+    updatedAt: Date.now() - 1000,
+  });
+
+  await createRejectableTransaction(knex, (trx) => migrateGranuleRecord(testGranule, trx));
+
+  const newerGranule = {
+    ...testGranule,
+    cmrLink: 'fakelink',
+  };
+
+  const granuleCumulusId = await createRejectableTransaction(
+    knex,
+    (trx) => migrateGranuleRecord(newerGranule, trx, { migrateAndOverwrite: 'true' })
+  );
+  const record = await granulePgModel.get(knex, {
+    cumulus_id: granuleCumulusId,
+  });
+
+  t.teardown(async () => {
+    await t.context.granulePgModel.delete(t.context.knex, { cumulus_id: record.cumulus_id });
+  });
+
+  t.deepEqual(record.cmr_link, newerGranule.cmrLink);
```

Comment:
We're just using `cmr_link` as a static property that won't change between migrations, yes?

Reply:
Correct, except that in this case *we* made the change for the purpose of this test.

Reply:
Cool. Just making sure I wasn't missing something important about `cmr_link`